doc/application_workflow/webserving.md

```diff
@@ -0,0 +1,137 @@
+# Web-serving on vSphere Integrated Container Engine
+
+We take the Web-serving benchmark from CloudSuite (http://cloudsuite.ch/webserving/) as an example, to demonstrate how customers who are interested in the LEMP implementation of a cloud native web-serving application could deploy it on vSphere Integrated Container Engine 0.7.0 with Docker Compose. This demo has three tiers deployed on three containerVMs: an Nginx Web server, a Memcached server, and a MySQL database server. The Web server runs Elgg (a social networking engine) and connects the Memcached server and the database server through the network.
```

Comment:
In Loc's voting app example, he still has the version number. So I think I should also put it here.